README.md

```diff
@@ -1,13 +1,15 @@
 # Project Movies
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This site is about popular movies in the US.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+In this project I used HTML, CSS, JS, React, API, BEM and NPM. This is my first site with multi-page React. I used npm to instal some libraries that were needed for my project like loader and fontawesome. When I had problem I used google and stackoverflow. If I had more time I would done black level :)
+
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://sandra-shumanteva-movie-site.netlify.app/movie/724989
```

Comment:
Hi Sandra! I will do your PR this week. The project looks great and you have even tried the red level 👍